orderer/common/multichannel/registrar.go

```diff
@@ -113,16 +113,18 @@ type Registrar struct {
 // ConfigBlock retrieves the last configuration block from the given ledger.
 // Panics on failure.
 func ConfigBlock(reader blockledger.Reader) *cb.Block {
```

Comment:
Can you add a UT for this function?

Reply:
Yes, I will update the PR with UT

Reply:
The main intention of adding the UT was to cover the panic paths. Given that this function can take a mock as input, it should be straight forward to write this.

Reply:
Also, as you may want to make these changes into master as well. I'll suggest to brake these changes into two. One the original backporting PR as is and new changes in the master first.

Reply:
I corrected the UT now covering all error scenarios.  Please check.

Should i need to create a separate PR for master branch?

Reply:
https://github.com/hyperledger/fabric/pull/2635 created a separate PR for main